models/tokenlon/ethereum/tokenlon_v5_ethereum_amm_v1_trades.sql

```diff
@@ -0,0 +1,116 @@
+{{ config(
+    schema = 'tokenlon_v5_ethereum',
+    tags=['dunesql'],
+    alias = alias('amm_v1_trades'),
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash'],
```

Comment:
based on `trace_address` being forced null in all spells here, i assume this will then eventually feed into `dex.trades`, not `dex_aggregator.trades`?

if `dex.trades`, then please match unique keys on all other dex spells. you can find examples in uniswap directory.

if `dex_aggregator.trades`, then please match unique keys on other dex aggregator spells. you can find examples in oneinch directory.